test/extensions/filters/http/ext_proc/ext_proc_grpc_fuzz_helper.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include "envoy/config/core/v3/base.pb.h"
+#include "envoy/extensions/filters/http/ext_proc/v3alpha/ext_proc.pb.h"
+#include "envoy/service/ext_proc/v3alpha/external_processor.pb.h"
+#include "envoy/type/v3/http_status.pb.h"
+
+#include "source/common/common/thread.h"
+#include "source/common/grpc/common.h"
+
+#include "test/common/http/common.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/test_common/utility.h"
+
+#include "grpc++/server_builder.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+using envoy::config::core::v3::HeaderValue;
+using envoy::config::core::v3::HeaderValueOption;
+using envoy::extensions::filters::http::ext_proc::v3alpha::ProcessingMode;
+using envoy::service::ext_proc::v3alpha::CommonResponse;
+using envoy::service::ext_proc::v3alpha::HeaderMutation;
+using envoy::service::ext_proc::v3alpha::ImmediateResponse;
+using envoy::service::ext_proc::v3alpha::ProcessingRequest;
+using envoy::service::ext_proc::v3alpha::ProcessingResponse;
+using envoy::type::v3::StatusCode;
+
+const uint32_t ext_proc_fuzz_max_data_size = 1024;
```

Comment:
Please make these constants like `ExtProcFuzzMaxDataSize` in naming, to allow the global consts to be identified later.